releasenotes/notes/bump-embedded-python-and-ntplib-98cb03455391b36a.yaml

```diff
@@ -0,0 +1,10 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+other:
+  - Upgrade embedded python 2 to ``2.7.18`` and ntplib to ``0.3.4``
```

Comment:
> Upgrade embedded python 2 to ``2.7.18``

Wouldn't this be a duplicate of the release note here: https://github.com/DataDog/datadog-agent/pull/5509?

(Note: for Agent 6/7 releases, software definitions in `datadog-agent` have priority over the ones defined in `omnibus-software`, so the `python2` definition in `omnibus-software` is not used in this repo)

Reply:
Thanks for the details, I just removed reference to python from the release note.